src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return (days * 40 - 50);
+  }
+
+  if (days >= 3 && days <= 6) {
+    return (days * 40 - 20);
+  }
+
+  if (days < 3) {
+    return (days * 40);
+  }
```

Comment:
second rule of [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md). please fix it